packages/tqdm/tqdm.0.1/opam

```diff
@@ -0,0 +1,30 @@
+opam-version: "2.0"
+bug-reports:  "https://github.com/LaurentMazare/ocaml-tqdm/issues"
+homepage:     "https://github.com/LaurentMazare/ocaml-tqdm"
+dev-repo:     "git+https://github.com/LaurentMazare/ocaml-tqdm.git"
+maintainer:   "Laurent Mazare <lmazare@gmail.com>"
+authors:      [ "Laurent Mazare" ]
+
+build: [["dune" "build" "-j" jobs "-p" name]]
+
+depends: [
+  "base" {>= "v0.12.0" & < "v0.13"}
```

Comment:
```suggestion
  "base" {>= "v0.12.0"}
```
Is this package really not compatible with `base.v0.13`?

Reply:
Seems to work with v0.13 so the constraint has been removed, thanks.